tests/ui_tests/test_bundle_list_page.py

```diff
@@ -28,84 +72,228 @@ def _assert_bundle_info_value(attribute: str, actual_info: BundleInfo, expected_
     ), f"Bundle's {attribute} should be {expected_value}, not {actual_value}"
 
 
+# pylint: disable=redefined-outer-name
+@allure.step('Check bundle list is empty')
+def _check_bundle_list_is_empty(page: BundleListPage):
+    assert (
+        row_count := page.table.row_count
+    ) == 0, f'Bundle list should be empty, but {row_count} records was found'
+
+
+@allure.step('Check bundle is listed in table')
+def _open_bundle_list_and_check_info(page: BundleListPage, expected_info: BundleInfo):
+    """
+    Open bundle list page, check that exactly 1 row is presented and check it's info
+    """
+    page.open()
+    assert (
+        row_count := page.table.row_count
+    ) == 1, f'Bundle list should has exactly 1 record, but {row_count} was found'
```

Comment:
```suggestion
    ) == 1, f'Bundle list should have exactly 1 record, but {row_count} was found'
```

Reply:
"List" is singular, so why "have"?

Reply:
https://www.quora.com/Why-dont-we-use-%E2%80%9Cshould-has%E2%80%9D-with-the-third-person-singular-number

Reply:
Okay